internal/indexer/indexer.go

```diff
@@ -642,21 +764,21 @@ func (i *Indexer) indexReferenceToExternalDefinition(p *packages.Package, docume
 	})
 
 	rangeID, _ := i.ensureRangeFor(pos, definitionObj)
-	refResultID := i.emitter.EmitReferenceResult()
-	_ = i.emitter.EmitTextDocumentReferences(rangeID, refResultID)
-	_ = i.emitter.EmitItemOfReferences(refResultID, []uint64{rangeID}, document.DocumentID)
-
```

Comment:
@tjdevries was putting this back the solution to the one-way navigation issue we were seeing?

Reply:
I fixed this by doing the linking now

Reply:
( linkImportMonikersToRanges )